ha/setup/const.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+from enum import Enum
+
+
+class TIMEOUT_ACTION(Enum):
+    START = "start"
+    STOP = "stop"
+    MONITOR = "monitor"
+
+class RESOURCE(Enum):
+    HAX = "hax"
+    MOTR_CONFD = "motr-confd"
+    MOTR_IOS = "motr-ios"
+    MOTR_FREE_SPACE_MON = "motr-free-space-mon"
+    S3_SERVER = "s3server"
+    S3_BACK_CONS = "s3backcons"
+    S3_BACK_PROD = "s3backprod"
+    S3AUTH = "s3auth"
+    HAPROXY = "haproxy"
+    SSPL_LL = "sspl-ll"
+    MGMT_VIP = "mgmt-vip"
+    CSM_AGENT = "csm-agent"
+    CSM_WEB = "csm-web"
+    KIBANA = "kibana"
+    EVENT_ANALYSER = "event_analyzer"
+    SRV_COUNTER = "srv_counter"
+    M_BUS_REST = "mbus_rest"
+    UDS = "uds"
+
+TIMEOUT_OFFSET=5
+
+TIMEOUT_MAP = {
```

Comment:
We should not hardcode it but read from respective systemd resource.

Reply:
Updated code
def get_res_timeout(resource: str, action: str, systemd: str = None) -> int:

if systemd is None then it will get resource from given map else take from systemd

Reply:
The given map should have default value of 90 sec for all systemd resources while I see different values?

Reply:
updated default time